index.d.ts

```diff
@@ -32,8 +32,12 @@ export declare interface FileOptions {
   private?: boolean;
 }
 
+export declare interface RegisterOptions {
+  prefix?: string;
+}
+
 export declare interface App {
-  [namespace: string]: HandlerFunction;
```

Comment:
unable to get it, why type changed from **HandlerFunction** to **any**.
**any** isn't preferable